RELEASING.md

```diff
@@ -0,0 +1,89 @@
+# Releasing
+
+When there is a need to release a new version, we should create a release in two places: GitHub and npm. Follow the instructions below to first bump the version, then create a GitHub release and then publish a new version of the package in npm.
+
+To release the npm package under the `@fintraffic` npm scope, you need to have at least Member permissions in the [fintraffic](https://www.npmjs.com/org/fintraffic) npm organization
+
+Once you have access, make sure you're logged into npm on the command line:
+
+```shell
+npm login
+```
+
+## 1. Update the version in package.json
+
+> Currently pushing directly to `main` branch is not allowed due to branch protection rules so you can't directly push the commit generated by `npm version` into `main`. So we need to first create a release branch for getting the `package.json` version update merged via a pull request.
+>
+> If you have permissions to edit branch protection rules in GH, this can be done simpler by avoidin a separate PR and review/approval just for version bump commit if you temporarily remove the PR requirement for changes to `main`.
```

Comment:
I would rather advice to merge the changes via PR, but to skip the review/approval process by merging with admin-rights. (My guess is that editing the branch protection rules requires admin-rights anyways, and if not, **it should**).

Reply:
Good point. I'll change this part a bit.

There are also many other ways releasing could be configured to be a bit more automatic but I think this is good enough for now with just the instructions (after I make a small update). For example version bump commit and/or release could be done automatically by GitHub actions when publishing a "GitHub Release" or when merging to a separate release branch, but that would require more configuration and thinking of process.

Reply:
Yep totally agree.

Reply:
I pushed an update.